docs/commands/install.md

```diff
@@ -15,6 +15,10 @@ Usage
 - `--uds-email UDS_CONTACT_EMAIL`              Contact e-mail address
 - `--uds-firstname UDS_CONTACT_FIRSTNAME`      Contact first name
 - `--uds-lastname UDS_CONTACT_LASTNAME`        Contact last name
+- `--dro-email DRO_CONTACT_EMAIL`              Contact e-mail address
```

Comment:
We have added the "dro" options but kept the uds ones, is that intentional?

Reply:
yes, not removing for now till all the users shift to DRO

Reply:
so we have flag BAS_ACTION which is by default UDS. UDS or DRO will be installed depending on the flag